tokio/src/sync/once_cell.rs

```diff
@@ -149,6 +149,38 @@ impl<T> OnceCell<T> {
         }
     }
 
+    /// Creates a new `OnceCell` that contains the provided value.
+    ///
+    /// This is equivalent to `<OnceCell<T> as From<T>>::from(value)`.
```

Comment:
It isn't, actually, due to the tracing stuff. Perhaps just delete this line?

Reply:
Thanks for pointing out, I've removed it.